Ex4Test.java

```diff
@@ -0,0 +1,40 @@
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class Ex4Test {
+
+    @Test
+    public void testCountIslands(){
+        boolean[][] mapTiles = new boolean[4][4];

```

Comment:
You could also initialize array via literal shaped in a nice rectangle, which would make it more clear what the map looks like:

```
boolean[][] mapTiles = {
    { false, true,  false, false },
    { true,  true,  false, false },
...
};
```

But feel free to leave it as is.